app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -1,4 +1,4 @@
-= render(Shared::DashboardComponent.new(heading: heading, navigation_items: vacancy_links, link: { url: organisation_jobs_path, text: t("buttons.create_job") } ))
+= render(Shared::DashboardComponent.new(heading: t("organisation.show.heading_html", organisation: @organisation.name, email: @email), navigation_items: vacancy_links, link: { url: organisation_jobs_path, text: t("buttons.create_job") } ))
```

Comment:
Dashboard Component can also be refactored to have the heading in the template rather than a long attribute.

Reply:
the dashboard is used by publishers and jobseekers. in the case of jobseekers, we do not display the organisation